INSTALL/production/INSTALL.rst

```diff
@@ -0,0 +1,430 @@
+This document describes the installation and setup of HaaS on CentOS 7.0.
+HaaS should work on other distros, but is not well tested or supported.
+For development environments, see ``HACKING.rst``.
+
+The HaaS Service node
+=====================
+
+This section talks about what must be done on the server upon which HaaS runs.
+
+Prerequisite Software
+---------------------
+
+HaaS requires a number of packages available through the CentOS RPM
+repositories, as well as the EPEL repository. EPEL can be enabled via::
+
+    yum install epel-release
+
+Then, the rest of the packages can be installed via::
+
+    yum install libvirt bridge-utils ipmitool telnet httpd mod_wsgi python-pip qemu-kvm python-virtinst virt-install python-psycopg2 vconfig net-tools
+
+In addition, HaaS depends on a number of python libraries. Many of these are
+available as RPMs as well, but we recommend installing them with pip, since
+this will install the versions that HaaS has been tested with.  This is done
+automatically by the instructions below.
+
+Setting Up HaaS Database
+------------------------
+
+The only DBMS currently supported for production use is PostgreSQL. (SQLite is
+supported for development purposes *only*). There are many guides on the web
+which describe setting up PostgreSQL; in these instructions we assume that
+you've done so successfully.
+
+You need to create a postgresql user and database for HaaS. Once you've done so,
+the ``uri`` option in ``haas.cfg``'s ``database`` must be set to::
+
+        postgresql://<user>:<password>@<address>/<dbname>
+
+Where ``<user>`` is the name of the postgres user you created, ``<password>`` is
+its password, ``<dbname>`` is the name of the database you created, and
+``<address>`` is the address which haas should use to connect to postgres (In a
+typical default postgres setup, the right value is ``localhost``).
+
+
+The HaaS software itself can then be installed by running::
+
+    git clone https://github.com/CCI-MOC/haas
+    cd haas
+    sudo python setup.py install
+
+To create the database tables, first make sure ``haas.cfg`` is set up the way
+you need, including any extensions you plan to use, then (from the directory
+containing ``haas.cfg``) exectue::
+
+    haas-admin db create
+
+Disable SELinux
+---------------
+
+The setup described in this document runs into problems with SELinux. In the
+future, we hope to ship a set of SELinux security policies with HaaS, but for
+now the solution is to disable SELinux::
+
+    sudo setenforce 0
+
+Make sure SELinux is also disabled on startup. To do this [on
+CentOS/RHEL](https://wiki.centos.org/HowTos/SELinux), edit
+`/etc/selinux/config` to change:
+```
+SELINUX=enforcing
+```
+to
+```
+SELINUX=permissive
+```
+
+Create User
+-----------
+
+First create a system user ``haas_user`` with::
+
+  sudo useradd haas_user -d /var/lib/haas -m -r
```

Comment:
i don't have a preference whether the user is haas or haas_user, but it should be consistent